src/services/editor.service.tsx

```diff
@@ -131,6 +131,15 @@ export class EditorService extends AbstractService {
     if (!file) {
       return;
     }
+    
+    // Check if file is valid (only JSON and YAML are allowed currently) ----Change afterwards as per the requirement
+    if (
```

Comment:
Nice to have for future refactoring: We could have a constant file where we put theses kind of values, as it will be easier to manage by other contributors.

Reply:
@Amzani Should I add a config file or sth else?